_posts/0000-01-02-Rumball7.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
would like to add
It's very easy to make some words **bold** and other words *italic* with Markdown.